vuepress/docs/next/tutorials/create/mfe/epc.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebarDepth: 2
+---
+
+# Create an Entando Platform Capability 
+
+An Entando Platform Capability, or EPC, is a packaged capability in the form of a bundle that adds functionality to the platform such as additional UX controls in the App Builder. With an EPC, adding menu options, an API management page, or integrating a WCMS like Strapi from the App Builder is simplified. This tutorial demonstrates how to build an EPC from a simple React micro frontend bundle.
+
+## Prerequisites
+* A working instance of Entando
+* An existing [React MFE](react.md)
+
+
+## Convert the React Bundle into an EPC
+Working with the [React MFE Tutorial](react.md), the following steps convert the React bundle into an EPC by modifying the bundle descriptor and customizing the public path to serve static assets. 
+
+### Configure the Bundle discriptor
+Edit the bundle descriptor `entando.json` in the root bundle directory. Remove the `type` and `titles` attributes and add the code snippet below, replacing the `paths`, `label` and `url` with your values.
```

Comment:
It feels a little odd to say "remove the 'type'" when we're really changing the type of the MFE. It might be better to break up the steps here. 1) change the type, 2) remove the titles, 3) add the new fields. I think that would make the conversion steps clearer.